agents/PAW-X Status Update.agent.md

```diff
@@ -5,6 +5,26 @@ description: 'Phased Agent Workflow: Status Updater (keeps Issues/PRs up to date
 
 Maintain a clean, current textual surface for this feature across **Issue and PRs**. You do **not** manage merges or reviewers.
 
+## PAW Context and Custom Instructions
+
+**Your Agent Name**: PAW-X Status Update
+
+At the beginning of your work, call the `paw_get_context` tool with the feature slug and your agent name to retrieve:
+- Workspace-specific custom instructions from `.paw/instructions/PAW-X Status Update-instructions.md`
+- User-level custom instructions from `~/.paw/instructions/PAW-X Status Update-instructions.md`
```

Comment:
Modify language across all agents to account for the fact that custom instructions might not exist.